tunnel/wstuncli.go

```diff
@@ -155,6 +160,33 @@ func NewWSTunnelClient(args []string) *WSTunnelClient {
 		wstunCli.Proxy = proxyURL
 	}
 
+	if *cliport != "" {
+		portList := strings.Split(*cliport, ",")
+		var ClientPorts []string
+		for _, v := range portList {
+			if strings.Contains(v, "..") {
+				k := strings.Split(v, "..")
+				bInt, err := strconv.Atoi(k[0])
+				if err != nil {
+					log15.Crit(fmt.Sprintf("Invalid Port Assignment: %q %v", *cliport, err))
+					os.Exit(1)
+				}
+
+				eInt, err := strconv.Atoi(k[1])
+				if err != nil {
+					log15.Crit(fmt.Sprintf("Invalid Port Assignment: %q %v", *cliport, err))
+					os.Exit(1)
+				}
+				for n := bInt; n <= eInt; n++ {
```

Comment:
Would it make sense check if `bInt > eInt` and return an error (i.e) `8000..7000`